### ISSUE TITLE
Revise url.py pattern-style

### ISSUE THREAD

**Issue by VBoB13** [VBoB13/TeachAdmin#2]
### URL patterns
Since I have now figured out that URL-patterns can be _greatly_ simplified compared to the "put it all into one variable"-solution and the fact that I've now realized that it's way easier to access related models objects through the in-built ORM instead of sending all the Model.pk values through the url-tag all the time.
Thus, the idea is that for each model, there will be an
`include([Model name]patterns)`
in the app's url.py to keep it all cleaner.

In a nutshell - this issue is just a 'cleaning' issue :)

**Comment by VBoB13** [comment]
Fixed as of now. Will probably revise further in the future, but this will do for now 😄
